bin/discovery.rb

```
@@ -145,6 +148,29 @@ def collect_host_attributes(host)
   attributes
 end
 
+def collect_host_vib_list(host)
+  task = host.configManager.patchManager.ScanHostPatchV2_Task
+  vib_list = []
+  attempts = 1
+  until task.info.state  == "success" || attempts > 6 do
+    attempts += 1
+    sleep 5
+  end
+  if task.info.state == "success" 
```

Should you add some kind of log message if the state doesn't end in `success`? Right now it will just silently return an empty array. You could raise an exception here and your handler would print it.